templates/frag/fromJson.hbs

```diff
@@ -13,8 +13,10 @@ Output a reference to a FromJsonFunction function.
 {{prefix}}stringFromJson
 {{~else if (isBinary)~}}
 {{prefix}}binaryFromJson
+{{~else if (isFile)~}}
+{{prefix}}fileFromMultipart
 {{~else if (isObject)~}}
-{{prefix}}model{{className nativeType.parentType}}FromJson
+{{prefix}}model{{className nativeType.parentType}}From{{#if (isSchemaSupportsMultipart .)}}MultipartFormData{{else}}Json{{/if}}
```

Comment:
I feel like this logic might not be right... if we've received a JSON request body then we want to parse it as JSON no matter whether the schema _supports_ multipart, such as in the case where one schema could be received as multipart or JSON.

My comment above about splitting `fromJson.hbs` into `fromJson.hbs` and `fromMultipart.hbs` will fix this I think.